spec/requests/search_results_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Search Results Endpoint', type: :request do
```

Comment:
It can be simplified to:

```suggestion
RSpec.describe 'Search Results', type: :request do
```